web/src/views/DandisetLandingView/ShareDialog.vue

```diff
@@ -6,7 +6,10 @@
     max-width="500"
   >
     <template #activator="{ on }">
-      <div v-on="on">
+      <div
+        style="display: inline"
```

Comment:
The `d-inline` class is a more idiosyncratic Vuetify way to do this.

```suggestion
        class="d-inline"
```